docs/source/app-dev/command-deduplication.rst

```diff
@@ -0,0 +1,352 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _command-deduplication:
+
+Command deduplication
+#####################
+
+The interaction of a Daml application with the ledger is inherently asynchronous: applications send commands to the ledger, and some time later they see the effect of that command on the ledger.
+Many things can fail during this time window: the application can crash, the participant node can crash, messages can be lost on the network, or the ledger may be just slow to respond due to a high load.
```

Comment:
"ledger may be just slow to load" - the word "just" is not needed

Reply:
Removed.